tests/PySys/tedge/tedge_software_update/run.py

```diff
@@ -1,13 +1,7 @@
 from environments.environment_c8y import EnvironmentC8y
 
 """
-Validate command line option help
-
-Given a running system
-When we call tedge help
-Then we find the string USAGE: in the output
-Then we find the string FLAGS: in the output
-Then we find the string SUBCOMMANDS: in the output
+TODO : Update this tests docu
```

Comment:
Might be better to do it in this PR.

Reply:
You convinced me, I reverse engineered the test docu.